### PR TITLE
Seed data

### PR DESCRIPTION
**Pull request by katbow**
Adds data using [faker](https://github.com/igas/faker/blob/master/USAGE.md). #339 

Demo data will include: 
200 service users, each with 1 story, tip, goal, and coping strategy.
10 clinicians connected to 10 service users each
20 carers connected to 10 service users each

Relationship migration & schema are added in order to create relationships between clinicians & service user, and carers & service users

Adds instructions for running a seed

### CODE REVIEW

**Comment by tonydaly** [comment]
Looks good! Maybe look into using [faker](https://github.com/igas/faker/blob/master/USAGE.md) for some more randomised data.

**Comment by nelsonic** [comment]
@katbow this looks good. 
As discussed this morning, we need a _lot_ more _realistic_ "demo data" to show off the functionality to clinicians and make it _look_ as realistic as possible for demos.
we _can_ merge this in now because it _works_, but ideally we should invest a bit more time to create a _lot_ more demo data.
I've added a comment to the original issue: https://github.com/healthlocker/healthlocker/issues/339#issuecomment-289810630

File: mix.exs [hunk 1]
```diff
@@ -44,7 +44,8 @@ defmodule Healthlocker.Mixfile do
      {:segment, github: "stueccles/analytics-elixir"},
      {:bamboo, "~> 0.7"},
      {:bamboo_smtp, "~> 1.2.1"},
-     {:mock, "~> 0.2.0", only: :test}]
+     {:mock, "~> 0.2.0", only: :test},
+     {:faker, "~> 0.7"}]
```
Comment (nelsonic) [inline mix.exs]: we _probably_ shouldn't load `faker` in Production ...? 🤔

File: priv/repo/seeds.exs [hunk 2]
```diff
@@ -9,3 +9,151 @@
...
+      password_hash: Comeonin.Bcrypt.hashpwsalt("password"),
+      name: Faker.Name.first_name(),
+      phone_number: Faker.Phone.EnGb.number(),
+      security_question: "Name of first boss?",
+      security_answer: "Betty",
+      data_access: Enum.random([true, false, nil]),
+      role: "support"
+    })
+
+    query = from u in User,
```
Comment (nelsonic) [inline priv/repo/seeds.exs]: does this query _guarantee_ to distribute the users over the carers _evenly_?

**Pull request merged**